ckanext/subscribe/fanstatic/subscribe.css

```diff
@@ -13,7 +13,16 @@
     margin-bottom: 5px;
 }
 #subscribe-form input {
-    width: 90px;
+    display: block;
+    margin-bottom: 5px;
+    padding: 6px 6px 6px 6px;
+    width: auto;
```

Comment:
The width of the input looks odd on mobile screen sizes.
![Screen Shot 2020-09-23 at 8 55 05 PM](https://user-images.githubusercontent.com/4096633/94088697-23e58480-fddf-11ea-9ae6-e108e625134c.png)

Reply:
I've changed the css for both the input field and button to be `-webkit-fill-available`